model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/state_utils/states.py

```diff
@@ -141,6 +141,14 @@ class MetricStateNonHydro:
     coeff_gradekin: gtx.Field[gtx.Dims[dims.ECDim], float]
 
 
+@dataclasses.dataclass
+class ConstantFieldsState:
```

Comment:
And by the way, why the name? If they are constant, then do we really need to serialize them?

Reply:
because I didn't have a clear idea of how to add new serialized data.
It needs to be serialized because it's not read from file but generated inside icon (but I've removed it from here)

Reply:
Maybe the name "constant"  is a bit misleading. It is constant throughout the simulation, but not constants such as the earth radius.
I think this `ConstantFieldsState` is better to be moved to somewhere in `common` as they only appear in computation of some metric fields as far as I know (vertical coordinates and horizontal gradients of topography, and these fields are used not only in dycore, they may also be used in physics such as turbulence).

Reply:
I'll move to `common/external_parameters` then, that's the nomenclature in both cosmo and icon for these fields.

Reply:
@halungge this needs the data to be updated again for the CI, could you please do that? same link
https://polybox.ethz.ch/index.php/s/IiRimdJH2ZBZ1od